config/services.yml

```diff
@@ -1148,3 +1117,34 @@ https://cmr.uat.earthdata.nasa.gov:
         operations: ['reformat', 'concatenate']
         conditional:
           format: ['application/x-zarr']
+
+  - name: harmony/service-example
+    description: |
+      Reference service that can be used to perform most operations supported by Harmony.
+      Useful for testing new API features end-to-end on example data but not meant for production use.
+    data_operation_version: '0.17.0'
+    type:
+      <<: *default-turbo-config
+      params:
+        <<: *default-turbo-params
+        env:
+          <<: *default-turbo-env
+          STAGING_PATH: public/harmony/service-example
+    umm_s:
+      - S1237980031-EEDTEST
```

Comment:
Based on Owen's comment that the service record is for netcdf-to-zarr we need to fix this to create a service-example service record and use that instead. We should also sanity check that no UMM-S record shows up in more than one service chain.

Reply:
Created new umm-s records for service-example in PROD and UAT. Verified no duplicate umm-s records in different service chains.

Reply:
I'll rebuild my local Harmony instance and run the affected regression test suites now.

Reply:
I re-ran the Trajectory Subsetter, Regridder and Swath Projector regression tests and they were sent along to the correct service now. Nice!